content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -21,21 +23,25 @@ const ActivityFeedItem = React.createClass({
 
 ActivityFeedItem.propTypes = {
   url: React.PropTypes.string.isRequired,
-  image: React.PropTypes.string, // TODO: Have fallback for this, change to icon
+  images: React.PropTypes.array,
   title: React.PropTypes.string.isRequired
 };
 
 const ActivityFeed = React.createClass({
+  getDefaultProps() {
+    return {length: 3};
```

Comment:
Not sure if this counts as a magic number, and we should convert to a const at the top of the file for easier readability.
ie: `const DEFAULT_ACTIVITY_FEED_LENGTH = 3;`